src/DGtal/io/Display3D.ih

```diff
@@ -808,10 +808,12 @@ DGtal::Display3D< Space ,KSpace >::addCone(const RealPoint &p1, const RealPoint
   zab = p2[2]-p1[2];
 
   norm = sqrt( xab*xab + yab*yab + zab*zab);
-  assert (norm > 0);
```

Comment:
when norm=0, you are trying to draw an cone with height=0.
imo this should be ignored and certainly not trigger any assert or error.
my understanding of this assert is that it guards the subsequent division by zero from generating nan. this can be easily avoided by placing the division inside a 'if(norm>0' block.